ci/docker/debian-10-cpp.dockerfile

```diff
@@ -69,7 +70,10 @@ RUN apt-get update -y -q && \
         tzdata \
         zlib1g-dev && \
     apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    wget -O /usr/local/bin/minio \
```

Comment:
We have an `ci/scripts/install_minio.sh` script which we could reuse here.

Reply:
Thanks!
I didn't notice it!